index.html

```diff
@@ -13,13 +13,15 @@
 		<meta property="og:image" content="assets/images/pyscript-preview.png">
 
     <link rel="icon" href="/assets/images/favicon.ico">
-		<link rel="apple-touch-icon" href="/assets/images/apple-touch-icon.png">
+	<link rel="apple-touch-icon" href="/assets/images/apple-touch-icon.png">
 
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
 
-		<link rel="stylesheet" href="assets/css/main.css?v=1.5">
+    <link rel="stylesheet" href="/assets/css/main.min.css?v=1.6">
+
+    <script src="assets/js/mailerlite-script.min.js?=v1.0" defer></script>
```

Comment:
nit: any particular reason this asset is not absolute like the css one, or vice-versa?

Reply:
Thanks @WebReflection, I pushed a change to make them all absolute.